### PR TITLE
Merging the Weekly suites

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Merging the Weekly suites

```
Merged below suites :
#  1.tier-3_cephfs_test_cephfs-top.yaml
#  2.tier-4_cephfs_recovery.yaml#
#  3.tier-2_cephfs_test-metrics.yaml
#  4.tier-3_cephfs_test_performance.yaml
#  5.tier-2_cephfs_test_mds_pinning.yaml
#  6.tier-2_cephfs_generic.yaml
#  7.tier-1_fs_kernel_pipeline.yaml 
```

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QTIPKJ/ 
we are seeing 10 failures all are known issues and product bugs.

After running I have verified cluster state it is in HEALTH_OK

I have missed fs_kernel re-running on the same setup

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml [hunk 1]
```diff
@@ -0,0 +1,393 @@
+---
+#=======================================================================================================================
+# Tier-level: 3
+# Test-Suite: tier-3_fs_top_recovery_metrics_perf_pin_generic_kernel
+# Conf file : conf/pacific/cephfs/tier-2_cephfs_9-node-cluster.yaml
```
Comment (sumabai) [inline suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml:5]: adding squid path would be good

File: suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+      polarion-id: CEPH-11267
+      name: "Cephfs Scrubing"
+  - test:
+      name: snap_retention_service_restart
+      module: snapshot_clone.snap_schedule_retention_vol_subvol.py
+      polarion-id: CEPH-83575627
+      desc: Verify Snapshot retention works even after service restarts
+      abort-on-fail: false
+      config:
+        test_name: snap_retention_service_restart
```
Comment (sumabai) [inline suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml:242]: Please add corresponding BZ id as comment for the failure seen

File: suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml (hunk 1 above)
Comment (hkadam134) [reply suites/squid/cephfs/tier_fs_top_recover_metric_perf_pin_generic_kernel.yaml:5]: +1

**Review by hkadam134 — APPROVED** [review]
/lgtm

**Review by sumabai — APPROVED** [review]

**Review by tintumathew10 — APPROVED** [review]
LGTM

**Pull request merged**